codespan-reporting/src/term/renderer.rs

```diff
@@ -127,6 +127,14 @@ impl<'writer, 'config> Renderer<'writer, 'config> {
         &self.config.styles
     }
 
+    pub fn start_context_lines(&self) -> usize {
+        self.config.start_context_lines
+    }
+
+    pub fn end_context_lines(&self) -> usize {
+        self.config.end_context_lines
+    }
```

Comment:
Let's add a comment here, just so I remember:
```suggestion
    // FIXME: Architectural smell - we'd prefer not to pass information from the renderer to the view.
    pub fn start_context_lines(&self) -> usize {
        self.config.start_context_lines
    }

    // FIXME: Architectural smell - we'd prefer not to pass information from the renderer to the view.
    pub fn end_context_lines(&self) -> usize {
        self.config.end_context_lines
    }
```